.gitignore

```diff
@@ -49,3 +49,6 @@ nbproject
 
 # temp
 .temp
+
+auth.json
+composer.lock
```

Comment:
is there a specific reason for adding composer.lock? 
as far as I know didn't we add it on purpose, as there might be projects when you want to add composer.lock to versioning. composer.lock is also not in gitignore of skeleton

Reply:
hey @fashxp 

I did not know that it was omitted with a reason, will remove it from .gitignore